CHANGELOG.md

```diff
@@ -10,12 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+
 - Replaces page type redirects, a.k.a. `/account`, `/login` to a corresponding file in `/pages` folder
 - Replaces `let` declarations for `useRef` for better React compatibility
+- Refactors cart sidebar
```

Comment:
Maybe we can give more details here, what do you think?

```suggestion
- Refactors cart sidebar using `Card` and `List` from `@faststore/ui`
```

Reply:
The problem is that it was more than this. I've also worked to remove the fixed height, to change how the HTML was organized, and to remove some classes. Don't know how to wrap those on the changelog.